### PR TITLE
add Rapa Nui and Uzbek resources

### PR DESCRIPTION
**Pull request by chrisgarrity**
Updating legacy strings for https://github.com/LLK/scratch-l10n/issues/80 and https://github.com/LLK/scratch-l10n/issues/79

### CODE REVIEW

File: legacy/documentation/rap/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1976,7 +1977,7 @@ msgstr ""
 #: templates/help/studio/tips/howto/wedo2win-intro.html:30 tips_config.py:257
 #: tips_config.py:267
 msgid "Troubleshooting"
-msgstr ""
+msgstr "Resuelve tus problemas"
```
Comment (benjiwheeler) [inline legacy/documentation/rap/LC_MESSAGES/django.po]: Just to make sure I'm following, is the idea that Rapa Nui has a combination of native words and Spanish words, from being a part of Chile?

**Comment by benjiwheeler** [comment]
What does this change in terms of Uzbek? And, where does it add these languages to the menu? I see the new translations for Rapa Nui, but not where Rapa Nui or Uzbek are added to any list.

File: legacy/documentation/rap/LC_MESSAGES/django.po (hunk 1 above)
Comment (chrisgarrity) [reply legacy/documentation/rap/LC_MESSAGES/django.po]: Yes, they're using Spanish if there isn't a Rapa Nui translation

**Comment by chrisgarrity** [comment]
`uz.po` files are already checked into legacy, so there was nothing that needed to be updated. Uzbek was a language that was included in Scratch 2.0. The list of languages is updated in scratchr2, but that should also pull in a new version of this repo, so it's waiting on this PR.

**Comment by benjiwheeler** [comment]
> uz.po files are already checked into legacy, so there was nothing that needed to be updated. Uzbek was a language that was included in Scratch 2.0. The list of languages is updated in scratchr2, but that should also pull in a new version of this repo, so it's waiting on this PR.

Got it. Thanks!

LGTM.

**Pull request merged**